lib/plugins/wallet/cardano/client.js

```diff
@@ -0,0 +1,19 @@
+const CardanoAPI = require('./api')
+
+const axios = require('axios')
+const https = require('https')
+const fs = require('fs')
+
+const coinUtils = require('../../../coin-utils')
+
+const cryptoRec = coinUtils.getCryptoCurrency('ADA')
+const cryptoDir = coinUtils.cryptoDir(cryptoRec)
+
+const httpsAgent = new https.Agent({
+  ca: fs.readFileSync(`${cryptoDir}/tls/client/ca.crt`),
```

Comment:
Can we assume that `cryptoDir` is a safe path? (I presume so but I want to check that an attacker cannot inject something like `../../` into it)

Reply:
Crypto dir is composed by Lamassu server itself, so if it will not be safe, then attacker is on server.

```
function cryptoDir (cryptoRec) {
  const code = cryptoRec.code
  return path.resolve(blockchainDir(), code)
}
```

&

```
function blockchainDir () {
  return options.blockchainDir
}
```

so there are two possible ways - `cryptoRec.code` which is get from `coin-utils.js` file, and `options.blockhainDir` which is get form `~/.lamassu/lamassu.json` - for me it seems that this path is safe (if server is safe).